### PR TITLE
Address Management

### PR DESCRIPTION
**Pull request by foolhardy21**
## Address Management.

This PR has address management that can be used on checkout page after placing order from the cart and search products feature on the products page.

- Files to review:
   - [ ] src/components/Profile/AddressCard.js
   - [ ] src/components/Profile/AddressForm.js
   - [ ] src/components/Profile/AddressesSection.js
   - [ ] src/contexts/address.context.js
   - [ ] src/pages/Profile.js
   - [ ] src/components/Checkout/CheckoutAddressCard.js
   - [ ] src/contexts/checkout.context.js
   - [ ] src/pages/Checkout.js
 
- Use this link for preview - https://deploy-preview-9--pensive-perlman-311072.netlify.app/login

v-1
v-2
Thanks

### CODE REVIEW

**Review by MayankKumar10 — APPROVED** [review]
Everything looks good Nice UI, nothing to suggest you can merge it.

File: src/contexts/address.context.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                addressState,
+                addressDispatch,
+                addressForm,
+                setAddressForm,
+                addressToBeUpdated,
+                setAddressToBeUpdated,
+            }}
+        >
+            {children}
+        </AddressContext.Provider>
```
Comment (anik31) [inline src/contexts/address.context.js:52]: appreciation - I liked the way you've implemented address management

**Review by anik31 — APPROVED** [review]
Nice work @foolhardy21 , everything looks good. The functionality is working perfectly in the preview

suggestion - you can add a check for empty input

File: src/contexts/address.context.js (hunk 1 above)
Comment (foolhardy21) [reply src/contexts/address.context.js:52]: thankyou

**Pull request merged**